.github/startup-test/docker-compose.yml

```diff
@@ -7,22 +7,32 @@ services:
         environment:
```

Comment:
Please move the file back. The file is used in the productive setup, so it is not suited to be in the `.github` folder.